portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,77 @@
+import CommentHandler from '/CommentHandler.js';
+import UserHandler from '/UserHandler.js';
+
+export default class Controller {
+
+  constructor(document) {
```

Comment:
See other comments about creating a this.commentHandler object in the constructor. The point is that the Controller just points around to its children objects (ie CommentHandler, UserHandler, others that may come up) and they do whatever work is applicable to them. The controller doesn't do any work, it just makes sure whichever class that is supposed to do the work will do the work at the right time.

Reply:
Get it, thanks!